Jenkinsfile

```diff
@@ -12,6 +12,28 @@ CQZ_BUILD_ID = new Date().format('yyyyMMddHHmmss')
 def jobs = [:]
 def helpers
 
+def withRVM(version, cl) {
```

Comment:
how about putting it in our shared library ? https://github.com/cliqz-oss/jenkins-pipeline-library